src/main/java/frc/robot/Constants.java

```diff
@@ -272,5 +277,27 @@ private static final AprilTagFieldLayout loadTransformedAprilTagFieldLayout() {
       loadTransformedAprilTagFieldLayout();
 
     public static final double kMaxX = kAprilTagFieldLayout.getFieldLength();
+
+    private static Translation2d getAprilTagTranslation(AprilTagFieldLayout fieldLayout, int aprilTagID) {
+      for (AprilTag aprilTag : fieldLayout.getTags()) {
+        if (aprilTag.ID == aprilTagID) {
+          return new Translation2d(aprilTag.pose.getX(), aprilTag.pose.getY());
+        }
+      }
+      assert(false);
+      return new Translation2d();
+    }
+
+    // The two april tag ID's used have to be across from eachother with different x coordinates
+    private static Translation2d calculateReefCenter(int aprilTagID1, int aprilTagID2) {
+      Translation2d aprilTag1 = getAprilTagTranslation(kAprilTagFieldLayout, aprilTagID1);
+      Translation2d aprilTag2 = getAprilTagTranslation(kAprilTagFieldLayout, aprilTagID2);
+      Translation2d centerOfReef = new Translation2d(
+        (aprilTag1.getX() + aprilTag2.getX()) / 2, aprilTag2.getY());
```

Comment:
This only works if the chosen AprilTags have the same y coordinate. This is brittle without needing to be. Let's allow for any opposing pair of AprilTags (and update the comment accordingly).

Also, divide by `2.0` for extra paranoia.